app/views/templates/ChangeSuccessView.scala.html

```diff
@@ -23,34 +24,48 @@
 
 @(model: ChangeSuccessViewModel)(implicit messages: Messages, appConfig: config.AppConfig, user: User[_])
 
+@buttonMessageKey = @{s"changeSuccess.viewBusinessDetails${if(user.isAgent){".agent"}else{""}}"}
+
 @mainTemplate(pageTitle = messages(model.pageTitle)) {
 
-  <div class="govuk-box-highlight">
-    <h1 class="govuk-heading-xl">@messages(model.pageTitle)</h1>
-  </div>
+<div class="govuk-panel govuk-panel--confirmation">
+    <h1 class="govuk-panel__title">
+        @messages(model.pageTitle)
+    </h1>
+</div>
 
   <h2 class="govuk-heading-m">@messages("common.whatHappensNext")</h2>
 
   @if(user.isAgent) {
 
     @model.agentEmail match {
-      case Some(email) => {<p class="govuk-body">@messages("changeSuccess.helpOne.digital.agent", email)</p>}
-      case _ => {<p class="govuk-body">@messages("changeSuccess.helpOne.paper.agent")</p>}
+      case Some(email) => {<p class="govuk-body">@messages("changeSuccess.helpOne.digital")</p>}
+      case _ => {
+        <p class="govuk-body">@messages("changeSuccess.helpOne.paper.agent")</p>
+}
```

Comment:
Could you just pop all this onto one line to be consistent with the case above it